atomisticparsers/gromacs/parser.py

```diff
@@ -712,6 +725,11 @@ def get_composition(children_names):
                 continue
             positions = self.traj_parser.get_positions(n)
             sec_system = sec_run.m_create(System)
+            sec_system.time = self.traj_parser.get_time(n)  # TODO Physical times should not be stored for GeometryOpt
```

Comment:
we can probably have a mapping here, from system index to time. 
```
system_time_map = {self.traj_parser.get_time(n) for n in range(len(sec_system))}
```

Reply:
done (I kept it inside the system loop though)